{"uuid": "7016068e-ce04-4316-9453-912683feca40", "children": ["bd00c817-f764-4e24-9450-5d8dc4baf6ac"], "befores": [{"name": "function_fixture2", "status": "passed", "start": 1705657041910, "stop": 1705657050611}], "afters": [{"name": "function_fixture2::driver_teardown", "status": "broken", "statusDetails": {"message": "AttributeError: 'WebDriver' object has no attribute 'close_app'\n", "trace": "  File \"/Users/hpc/pytest/venv/lib/python3.10/site-packages/allure_commons/_allure.py\", line 221, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/Users/hpc/pytest/conftest.py\", line 25, in driver_teardown\n    request.instance.driver.close_app()\n"}, "start": 1705657072354, "stop": 1705657072355}], "start": 1705657041910, "stop": 1705657072355}